tests/Aspire.Hosting.Tests/WithEndpointTests.cs

```diff
@@ -490,6 +492,39 @@ public async Task VerifyManifestProjectWithHttpEndpointDoesNotAllocatePort()
         Assert.Equal(expectedManifest, manifest.ToString());
     }
 
+    [Fact]
+    public async Task VerifyManifestProjectWithEndpointsSetsPortsEnvVariables()
```

Comment:
I would move this to project tests

Reply:
But then shouldn't the one above it (`VerifyManifestProjectWithHttpEndpointDoesNotAllocatePort`) be moved too? They seem similar in spirit, and they share the `TestProject` inner class.

Reply:
Yep they should both be moved.

Reply:
Spoke offline, I'll move these in a later refactoring.